### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by emmyobonyo**
- Implemented Calculator UI using Class components.
- Deleted unnecessary files and code blocks created by ```create-react-app```.
- Performed CSS and ES linter to ensure best practices.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,57 @@
...
+.calculator-button {
+  background-color: lightgray;
+  border: 1px solid black;
+  padding: 20px;
+  font-size: 30px;
+  text-align: center;
+}
+
+.orange-button {
+  background-color: orange;
```
Comment (henatan99) [inline src/App.css:39]: - [ ] I recommend you to stick with the original design. This color doesn't represent the original color and you can use a color picker chrome extension to pick the color from the original design. Let's assume the original design is a client's requirement and we need to stick with it as much as possible. :pray: 

image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @emmyobonyo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] There is a typo in your project repo name. Kindly fix it as it gives a wrong impression about your professionalism. :pray: 
- [x] I think your PR summary is descriptive enough. It should reflect the main changes you made in this branch. Kindly make it more descriptive. :pray: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,57 @@
...
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.calculator {
+  width: 50%;
+  margin: auto;
+  margin-top: 50px;
+  border: 3px solid black;
 }
```
Comment (EricMbouwe) [inline src/App.css:10]: - [ ]  Let's remove these borders to match the given design.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @emmyobonyo,

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @emmyobonyo, 

# Status: Approved✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**